EPlast/EPlast.BLL/Interfaces/Club/ClubMembers.cs

```diff
@@ -0,0 +1,10 @@
+﻿using EPlast.BLL.DTO.Club;
+
+namespace EPlast.BLL.Interfaces.Club
+{
+    public interface ClubUser
```

Comment:
Seems in all projects we use the "I" prefix for interfaces
Shouldn`t be here? 
```suggestion
    public interface **IClubUser**
```